tests_new/integration_tests/README.md

```diff
@@ -64,24 +64,87 @@ Currently **we support only Cognito based deployments** but support for any IdP
                 "accountId": "...",
                 "region": "eu-west-1"
               },
-               "persistent_env1": {
-                       "accountId": "...",
-                       "region": "us-east-1"
-               },
+              "persistent_env1": {
+                "accountId": "...",
+                "region": "us-east-1"
+              },
+               "persistent_cross_acc_env_1": {
+                  "accountId": "...",
+                  "region": "us-east-1"
+                },
               "session_cross_acc_env_1": {
-                       "accountId": "another acc",
-                       "region": "same as session_env1"  
-               },
+                "accountId": "...",
+                "region": "eu-central-1"
+              }
             },
             "dashboards": {
                 "session_env1": {
                   "dashboardId": "..."
                 },
+              },
+             "redshift_connections": {
+              "connection_serverless_admin_session_env1": {
+                "namespace_id": "...",
+                "workgroup": "...",
+                "secret_arn": "..."
+              },
+              "connection_serverless_data_user_session_env1": {
+                "namespace_id": "...",
+                "workgroup": "...",
+                "secret_arn": "..."
+              },
+              "connection_cluster_admin_session_cross_acc_env_1": {
+                "cluster_id": "...",
+                "secret_arn": "..."
+              },
+              "connection_cluster_data_admin_session_cross_acc_env_1": {
```

Comment:
this needs to be `connection_cluster_data_user_session_cross_acc_env_1` not `connection_cluster_data_admin_session_cross_acc_env_1`

Reply:
Done